### DISCUSSION TITLE
Start Page with bookmarks listed

### DISCUSSION THREAD

**Discussion by pawelrochala** [plateaukao/einkbro#175] [Ideas]
Hi 
It would be a nice feature to list all bookmarks as the start page.

**Comment by plateaukao** [comment]
@pawelrochala 
You can configure what content should be displayed as start page (for new tabs) in Settings > UI > "New tab behavior".
As for the very first tab when launching EinkBro without saving previous tabs, this is a nice suggestion too. I will consider implement it in the future. Thanks.
